deformable_ompl/SimplifyPathQuery query
---
deformable_ompl/SimplifyPathResult result
